notebooks/tel_and_site/subsys_req_ver/mthexapod/SITCOM-2190_apply_fft_to_motor_current.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<ul><li>This is not obvious from the plot. Especially if you plot the 6 motor noises, it looks to me that motor 1 also have a strange behavior.</li><li>The fact that the current noise is high doesn't necessarily mean that there is a problem. What would point to a problem would be a sudden change in the noise level</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>